Kontent.Ai.Management/IManagementClient.cs

```diff
@@ -607,6 +607,13 @@ public interface IManagementClient
     /// <param name="scheduleModel">The time when the language variant will be unpublished</param>
     Task ScheduleUnpublishingOfLanguageVariantAsync(LanguageVariantIdentifier identifier, ScheduleModel scheduleModel);
 
+    /// <summary>
+    /// Schedules publish and unpublish of language variant.
+    /// </summary>
+    /// <param name="identifier">The identifier of the language variant that should be scheduled.</param>
+    /// <param name="schedule">The interval in which the variant should be published</param>
+    Task SchedulePublishAndUnpublishOfLanguageVariantAsync(LanguageVariantIdentifier identifier, SchedulePublishAndUnpublishModel schedule);
```

Comment:
nitpick, but for the sake of consistency, I would rename this to `SchedulePublishingAndUnpublishingOfLanguageVariantAsync`. the six extra symbols don't make that much of a difference and it would match the endpoint name in API docs :)

Reply:
Should be resolved. :)